include/global.h

```diff
@@ -181,6 +175,16 @@ struct Time
     /*0x04*/ s8 seconds;
 };
 
+
+struct SaveBlock3
+{
+#if OW_USE_FAKE_RTC
+	struct Time fakeRTC;
```

Comment:
spaces instead of tabs

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/4910/commits/957dda9d1f078ae2fec18f1a7eaf881795225cb6